VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionService.java

```diff
@@ -80,43 +81,43 @@ CourseDiscussionInfo getCourseDiscussionInfoWithCacheEnabled(@Path("course_id")
 
 
     @GET("/api/discussion/v1/threads/")
-    TopicThreads getThreadList(@Query("course_id") String courseId,
-                               @Query("topic_id") List<String> topicIds,
-                               @Query("view") String view,
-                               @Query("order_by") String orderBy,
-                               @Query("page_size") int pageSize,
-                               @Query("page") int page)
+    Page<DiscussionThread> getThreadList(@Query("course_id") String courseId,
+                                         @Query("topic_id") List<String> topicIds,
+                                         @Query("view") String view,
+                                         @Query("order_by") String orderBy,
+                                         @Query("page_size") int pageSize,
+                                         @Query("page") int page)
             throws RetroHttpException;
 
 
     @GET("/api/discussion/v1/threads/")
-    TopicThreads getFollowingThreadList(@Query("course_id") String courseId,
-                                        @Query("following") String following,
-                                        @Query("view") String view,
-                                        @Query("order_by") String orderBy,
-                                        @Query("page_size") int pageSize,
-                                        @Query("page") int page)
+    Page<DiscussionThread> getFollowingThreadList(@Query("course_id") String courseId,
+                                                  @Query("following") String following,
+                                                  @Query("view") String view,
+                                                  @Query("order_by") String orderBy,
+                                                  @Query("page_size") int pageSize,
+                                                  @Query("page") int page)
             throws RetroHttpException;
 
     @GET("/api/discussion/v1/threads/")
-    TopicThreads searchThreadList(@Query("course_id") String courseId,
-                                  @Query("text_search") String text,
-                                  @Query("page_size") int pageSize, @Query("page") int page)
+    Page<DiscussionThread> searchThreadList(@Query("course_id") String courseId,
+                                            @Query("text_search") String text,
+                                            @Query("page_size") int pageSize, @Query("page") int page)
```

Comment:
visual nit: param `@Query("page") int page` should be moved to next line to maintain the class formatting.